styles.css

```diff
@@ -90,6 +90,61 @@ button {
 
 nav {
   display: none;
+  flex-direction: column;
+  justify-content: space-between;
+  position: absolute;
+  width: 100vw;
+  height: 100vh;
+  top: 0;
+  right: 0;
+  background-color: #fff;
+}
```

Comment:
[OPTIONAL]
- [ ] You can also consider adding `position: fixed` to the mobile menu to prevent it scrolling with the page.